awx/ui/client/src/license/license.controller.js

```diff
@@ -111,14 +111,14 @@ export default
             const raw = new FileReader();
 
             raw.onload = function() {
-                $scope.newLicense.entitlement_cert = raw.result;
+                $scope.newLicense.manifest = btoa(raw.result);
```

Comment:
There's probably a better way to do this correctly, but it likely involves changing our POST content-type to have binary data in the content body *or* JSON and meeeeeeeeeeeeeh whatever this works fine.

Reply:
cc @mabashian